apps/cnquery/cmd/status.go

```diff
@@ -149,37 +149,50 @@ type ClientStatus struct {
 func (s Status) RenderCliStatus() {
 	if s.Client.Platform != nil {
 		agent := s.Client
-		log.Info().Msg("Platform:\t" + agent.Platform.Name)
-		log.Info().Msg("Version:\t" + agent.Platform.Version)
-		log.Info().Msg("Hostname:\t" + agent.Hostname)
-		log.Info().Msg("IP:\t\t" + agent.IP)
+		log.Info().Msg("Platform:\t\t" + agent.Platform.Name)
```

Comment:
why is this changed to a double tab?

Reply:
They looked more properly lined up this way for me.